### ISSUE TITLE
Different width for space and nbspace glyphs

### ISSUE THREAD

**Issue by khaledhosny** [adobe-fonts/source-sans-pro#96]
I’m not sure if this is intentional, but I think they should be the same (I can send a PR if they should indeed be the same).

**Comment by pauldhunt** [comment]
If this is the case, it is indeed a bug. I will track this for the next release.

**Comment by khaledhosny** [comment]
FWIW, I tried to fix this and make a PR, but after fixing the masters and running `makeInstancesUFO -d RomanMasters/SourceSansPro.designspace` I get a huge number of differences, probably something changed in the current versions of the tools.

**Comment by frankrolf** [comment]
That could simply be fixed in the UFO XML files, no? (Admittedly old-school, but at least no other files are changed). The big amount of changes likely stems from an update of the UFO normalizer – it now adds line breaks at the end of each document, whereas before it did not.

**Comment by khaledhosny** [comment]
Yes editing XML file is one way (I already did that), but then there is the `font.ttf` files which (IIRC) were generated by FontLab.

**Comment by frankrolf** [comment]
Good point. Well, if anything, at least this issue made it onto Paul’s radar. Thanks for bringing it up.

**Comment by miguelsousa** [comment]
@pauldhunt currently makeotf supports multi-mapping a glyph via the GOADB; the support was added sometime last year I believe.

I think it's better to support U+00A0 (nbspace) and other characters via multi-mapping because you avoid inconsistencies (different advance width, different outline, different OT features) and cut down the number of duplicate glyphs.

Here are the GOADB entries I used in a recent project,
```
space	space	uni0020,uni00A0
grave	grave	uni0060,uni02CB
acute	acute	uni00B4,uni02CA
macron	macron	uni00AF,uni02C9
Delta	Delta	uni0394,uni2206
Omega	Omega	uni03A9,uni2126
mu	mu	uni03BC,uni00B5
quoteleft	quoteleft	uni2018,uni02BB
quoteright	quoteright	uni2019,uni02BC
fraction	fraction	uni2044,uni2215
hyphen	hyphen	uni002D,uni00AD,uni2010
periodcentered	periodcentered	uni00B7,uni2219
```

**Comment by miguelsousa** [comment]
Fixed in version 2.040.
